### ISSUE TITLE
Not working on Github Actions CI

### ISSUE THREAD

**Issue by cayleyh** [lyft/set-simulator-location#32]
I'm trying to use `set-simulator-location` for testing behaviour on CI using [Detox](https://github.com/wix/Detox). It works fine when running locally, but does not work at all when running on Github Actions CI.

Has anyone encountered this? Is it a known limitation or is there some build configuration that is necessary to enable location setting on CI servers?

**Comment by keith** [comment]
Is the simulator app launched? if not you might try that

**Comment by cayleyh** [comment]
Yes it has launched & I've confirmed by finding the simulator with the same ID marked as 'booted' with `simctl`.

**Comment by cayleyh** [comment]
Any thoughts on things I can try to help debug this?

**Comment by triztian** [comment]
The tool relies on [DistributedNotificationCenter](https://developer.apple.com/documentation/foundation/distributednotificationcenter), I've seen some older reports that indicate that it might not be working, for example: https://developer.apple.com/forums/thread/129437

Not sure if your issue would be the same as that one but worth exploring around it.

**Comment by cayleyh** [comment]
Thanks @triztian. Theoretically the version of MacOS that is running the the Github Actions container is the same as what I'm running locally, but I'll check this out to see if anything others have reported here can help debug better.

**Comment by KatieSnell-TomTom** [comment]
I think I am noticing the same thing on my Detox setup, which is using TeamCity rather than Github Actions CI. Works locally, but not on the build system.

Any ideas what could be different?

**Comment by henriquenfaria** [comment]
Same issue on Azure Pipelines machines. Have anyone found a solution already?
